ui/tests/helpers/glimmer-factory.js

```diff
@@ -0,0 +1,25 @@
+// Look up the component class in the glimmer component manager and return a
+// function to construct components as if they were functions.
```

Comment:
Yeah, I'll add this to a comment as well as here but unit testing means I can access methods and properties directly rather than indirectly through DOM inspection. They also run faster since there's no rendering but that's not the primary motivator.